tests/test_classification.py

```diff
@@ -644,6 +644,7 @@ def test_xval_float_classes_as_strings():
     prediction_prefix = join(_my_dir, 'output', 'float_class')
     learner = Learner('LogisticRegression')
     learner.cross_validate(float_class_fs,
+                           grid_search=True,
```

Comment:
do we need `grid_search=True` if it is true by default?

Reply:
No we don't but I want people to start being explicit so it's good if our tests are too.